source/fundamentals/run-command.txt

```diff
@@ -55,17 +55,55 @@ the current member's role in the replica set, on a database:
 For a full list of database commands and corresponding parameters, see
 the :ref:`Additional Information section <addl-info-runcommand>`.
 
+.. _node-command-options:
+
+Command Options
+---------------
+
+You can specify optional command behavior for the ``command()``
+and ``runCursorCommand()`` methods.
+
+The ``command()`` method accepts a ``RunCommandOptions`` object. To learn
+more about the ``RunCommandOptions`` type, see the `API documentation <{+api+}/types/RunCommandOptions.html>`__.
+
+The ``runCursorCommand() method`` accepts a ``RunCursorCommandOptions``
+object. To learn more about the ``RunCursorCommandOptions`` type, see
+the `API documentation <{+api+}/types/RunCursorCommandOptions.html>`__.
+
+Starting in version 6.0 of the {+driver-short+}, you can pass only the
+following options directly to the ``command()`` method:
+
+- ``comment``
+- ``enableUtf8Validation``
+- ``raw``
+- ``readPreference``
+- ``session``
+
+You can set additional options, including a write or read
+concern, in the command document that you pass to
+the ``command()`` method. See the Server manual :manual:`entry
+</reference/command/>` for a command to learn about its associated
+options.
+
+The following code shows how to specify a write concern
+in the command document for a ``delete`` command:
+
+.. code-block:: javascript
+   :emphasize-lines: 4
+
+   const commandDoc = {
+       delete: "values",
+       deletes: [ { q: { x: "abc" }, limit: 0 } ],
+       writeConcern: { w: "majority", wtimeout: 5000 }
+   };
+   const result = await myDB.command(commandDoc);
+
 .. note:: Read Preference
 
-   ``command()`` and ``runCursorCommand()`` do not obey the read
-   preference you may have set on your ``Db`` object elsewhere in
+   The ``command()`` and ``runCursorCommand()`` methods do not execute with
+   the read preference you may have set on your ``Db`` object earlier in
```

Comment:
Suggestion:

I think it would be easier to understand and more concise by using [Positive Statements](https://www.mongodb.com/docs/meta/style-guide/writing/use-positive-statements/):

```suggestion
   The ``command()`` and ``runCursorCommand()`` methods ignore the
   read preference setting you may have set on your ``Db`` object.
```

I don't think it's necessary to mention "earlier in your code" since "you may have set" refers to the recent past.